porting/examples/linux/include/syscfg/syscfg.h

```diff
@@ -771,6 +783,14 @@
 #define MYNEWT_VAL_BLE_L2CAP_SIG_MAX_PROCS (1)
 #endif
 
+#ifndef MYNEWT_VAL_BLE_MAX_BIG
+#define MYNEWT_VAL_BLE_MAX_BIG (MYNEWT_VAL_BLE_MULTI_ADV_INSTANCES)
+#endif
+
+#ifndef MYNEWT_VAL_BLE_MAX_BIS
+#define MYNEWT_VAL_BLE_MAX_BIS (4)
```

Comment:
it's per BIG ? or total ?

Reply:
As for now, total. I have nothing against changing it to per BIG, I feel like both approaches are  valid

Reply:
total should be fine.